### PR TITLE
add sengled motion sensor

### PR DESCRIPTION
**Pull request by sengled-desi**
1.add the init.lua file to the path ~\SmartThingsEdgeDrivers\drivers\SmartThings\zigbee-motion-sensor\src\sengled;
2.Modified ~\SmartThingsEdgeDrivers\drivers\SmartThings\zigbee-motion-sensor\fingerprints.yml file, a new device type E1M-G7H is added.

### CODE REVIEW

File: drivers/SmartThings/zigbee-motion-sensor/src/sengled/init.lua [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      return true
+    end
+  end
+  return false
+end
+
+local function device_init(driver, device)
+  battery_defaults.build_linear_voltage_init(2.2, 3.0)(driver, device)
+
+  for _, attribute in ipairs(CONFIGURATIONS) do
```
Comment (ctowns) [inline drivers/SmartThings/zigbee-motion-sensor/src/sengled/init.lua:42]: @greens should a test_sengled_sensor.lua file be added to test that these configurations are set correctly? It looks like that is the unique behavior in this subdriver

Comment (greens) [reply drivers/SmartThings/zigbee-motion-sensor/src/sengled/init.lua:42]: that would be nice to see

**Comment by greens** [comment]
@sengled-desi please rebase your fork. There have been updates to the CI since your fork was created that will do some integral things for your WWST submission.

**Comment by lelandblue** [comment]
Hello @sengled-desi please rebase your fork as soon as your schedule allows, thank you!

**Comment by sengled-desi** [comment]
> Hello @sengled-desi please rebase your fork as soon as your schedule allows, thank you!

Hello @lelandblue    init.lua file is modified and submitted

**Comment by greens** [comment]
@sengled-desi looks like you weren't able to bring in the latest changes from the base repository.

You'll need our repo set up as a `git remote` (I'd recommend `git remote add remote [ST repo SSH URL]`) then you'll want to `git fetch remote main` and then `git rebase remote/main`.

Also I'll point out that you'll need to `require` your sub-driver from the base driver you've added it to (in the base `init.lua`) before any of your code will run. 

You will also need to write tests for your code.

**Comment by sengled-desi** [comment]
@greens    I need to add files under the main branch. If this is the case, I have added and submitted it from the new

**Comment by sengled-desi** [comment]
> @sengled-desi looks like you weren't able to bring in the latest changes from the base repository.
> 
> You'll need our repo set up as a `git remote` (I'd recommend `git remote add remote [ST repo SSH URL]`) then you'll want to `git fetch remote main` and then `git rebase remote/main`.
> 
> Also I'll point out that you'll need to `require` your sub-driver from the base driver you've added it to (in the base `init.lua`) before any of your code will run.
> 
> You will also need to write tests for your code.
@greens    hello,greens
this is my newly submitted address:  https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/664

**Comment by lelandblue** [comment]
I am going to close this PR since [](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/664 )is the replacement

**Pull request closed without merging**